### ISSUE TITLE
Fatal bug in Rstudio server 2024.04.0+735: crashes when auto complete triggers on dataframe with non-english variable names - with simple fix

### ISSUE THREAD

**Issue by emilwest** [rocker-org/rocker-versioned2#818]
### Container image name

rocker/tidyverse:4.4.0

### Container image digest

rocker/tidyverse@sha256:e1d49ecb6e41f794328bc61bf11b0d83f99964d656695971c7eb9271344c8261

### What operating system are you seeing the problem on?

Linux

### System information

- Docker version 26.1.0, build 9714adc
- Docker Compose version v2.13.0
- R version 4.4.0 (2024-04-24)
- Platform: x86_64-pc-linux-gnu
- Running under: Ubuntu 22.04.4 LTS

### Bug description

Rstudio server crashes when auto-complete triggers on a data frame using non-ascii characters in variable names such as å, ä, ö.
Here is an exampe of the error message appearing: 

![image](https://github.com/rocker-org/rocker-versioned2/assets/11556959/57fe75e1-9433-435f-bab6-42bc771bea26)

This was first detected in a github issue [here](https://github.com/rstudio/rstudio/issues/14625) using a dataframe containing column names written in japanese. In that thread there are more examples. 

It has since been fixed in RStudio Server 2024.04.1+748 and 2024.07.0-daily+102 on Ubuntu 22.

So a fix for this issue could be to simply update the rstudio version in the rocker image, that would be highly appreciated. Could a fix for this be implemented within the coming weeks? Cheers.

### How to reproduce this bug?

Log in to a Rstudio server session, open up a new R-script and type the following:

```R

library(tidyverse)

df <- tibble(ärendenummer = c("a"), omrade = 1)
df

df |> 
  # after the comma, trigger auto-complete with ctrl+space and the session will crash
  select(ärendenummer, )
```

**Comment by benz0li** [comment]
@emilwest Workaround:

```R

library(tidyverse)

df <- tibble(ärendenummer = c("a"), omrade = 1)
df

df |> select("ärendenummer", )
```

**Comment by eitsupi** [comment]
Do you think this is closed by #819?

**Comment by benz0li** [comment]
> Do you think this is closed by #819?

Yes.

**Comment by emilwest** [comment]
Thanks for such a quick fix! Can confirm that the bug no longer appears in the latest updated image rocker/tidyverse:4.4.0 on rstudio server 2024.04.1+748.
